README.md

```diff
@@ -1,255 +1,72 @@
 
-The below command expects the user to provide
-* A cluster wide private address range of $CLUSTER_IP_SUBNET
-(e.g: 192.168.0.0/16).  The pods are provided IP address from this range.
+## Run daemonsets
 
-* $NODE_NAME should be the same as the one used by kubelet.  kubelet by default
-uses the hostname.  kubelet allows this name to be overridden with
---hostname-override.
-
-* $SERVICE_IP_SUBNET is the same as the one provided to k8s-apiserver via
---service-cluster-ip-range option. An e.g is 172.16.1.0/24.
+On Kubernetes master, label it to run daemonsets.
 
 ```
- nohup sudo ovnkube -k8s-kubeconfig kubeconfig.yaml -net-controller \
- -loglevel=4 \
- -k8s-apiserver="http://$CENTRAL_IP:8080" \
- -logfile="/var/log/ovn-kubernetes/ovnkube.log" \
- -init-master=$NODE_NAME -init-node=$NODE_NAME \
- -cluster-subnet="$CLUSTER_IP_SUBNET" \
- -k8s-service-cidr=$SERVICE_IP_SUBNET \
- -nodeport \
- -init-gateways -gateway-local \
- -k8s-token="$TOKEN" \
- -nb-address="tcp://$CENTRAL_IP:6641" \
- -sb-address="tcp://$CENTRAL_IP:6642" 2>&1 &
+kubectl taint nodes --all node-role.kubernetes.io/master-
+kubectl label node $NODENAME node-role.kubernetes.io/master=true --overwrite
 ```
 
-Note: Make sure to read /var/log/ovn-kubernetes/ovnkube.log to see that there were
-no obvious errors with argument passing.  Also, you should only pass
-"-init-node" argument if there is a kubelet running on the master node too.
-
-If you want to use SSL instead of TCP for OVN databases, please read
-[INSTALL.SSL.md].
-
-### Minion node initialization.
-
-On each host, you will need to run the following command once.
-
-The below command expects the user to provide
-* A cluster wide private address range of $CLUSTER_IP_SUBNET
-(e.g: 192.168.0.0/16).  The pods are provided IP address from this range.
-This value should be the same as the one provided to ovnkube in the master
-node.
-
-* $NODE_NAME should be the same as the one used by kubelet.  kubelet by default
-uses the hostname.  kubelet allows this name to be overridden with
---hostname-override.
-
-* $SERVICE_IP_SUBNET is the same as the one provided to k8s-apiserver via
---service-cluster-ip-range option. An e.g is 172.16.1.0/24.
-
-* If you are using ingress controllers with L7 load-balancing to enter into
-the k8s cluster, you can skip providing the '-nodeport' option with the
-below command.
+Create OVN daemonset yamls from templates by:
+(The $MASTER_IP below is the IP address of the machine where kube-apiserver is
+running)
 
 ```
-nohup sudo ovnkube -k8s-kubeconfig kubeconfig.yaml -loglevel=4 \
-    -logfile="/var/log/ovn-kubernetes/ovnkube.log" \
-    -k8s-apiserver="http://$CENTRAL_IP:8080" \
-    -init-node="$NODE_NAME"  \
-    -nodeport \
-    -nb-address="tcp://$CENTRAL_IP:6641" \
-    -sb-address="tcp://$CENTRAL_IP:6642" -k8s-token="$TOKEN" \
-    -init-gateways \
-    -k8s-service-cidr=$SERVICE_IP_SUBNET \
-    -cluster-subnet=$CLUSTER_IP_SUBNET 2>&1 &
+# Clone ovn-kubernetes repo
```

Comment:
can you change it to download the `daemonset.sh` script and for the apply use it from git raw I think it should be more user friendly instead of cloning all the repo.

Reply:
@SchSeba
I am not sure I understand. When we run daemonset.sh, it takes in templates and puts in the yaml files into a different directory. So, just downloading daemonset.sh will not be enough as we need the directory structure and templates too. I agree that, cloning the repo is not ideal. But, we haven't had a agreement on what is the best way to create those default yaml files yet. Open to ideas.